pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -140,15 +140,8 @@ func (s *Server) initK8SConfigStore(args *PilotArgs) error {
 	configController := s.makeKubeConfigController(args)
 	s.ConfigStores = append(s.ConfigStores, configController)
 
-	var WaitForCRD func(class schema.GroupVersionResource, stop <-chan struct{}) bool
-	if s.kubeClient.CrdWatcher() == nil {
-		WaitForCRD = func(class schema.GroupVersionResource, stop <-chan struct{}) bool {
-			return true
-		}
-	} else {
-		WaitForCRD = func(class schema.GroupVersionResource, stop <-chan struct{}) bool {
-			return s.kubeClient.CrdWatcher().WaitForCRD(class, stop)
-		}
+	WaitForCRD := func(class schema.GroupVersionResource, stop <-chan struct{}) bool {
```

Comment:
I thought the reason we had this code was because istiod does not have permission to watch CRD resources.  Is this still true (or am I mistaken)?

Reply:
Rob, Yes. In the upstream, they stop scanning the CRD at starting the server, and they also introduced a `DelayedInformer`, which works closely with the `CrdWatcher`. I move the CRD related logic into `CrdWatcher` in this [commit](https://github.com/maistra/istio/pull/920/commits/e94bf1c1995c3d1069943c7bc775ebc09536b9b2).

Reply:
Sorry but I will push to not add any refactor to the patched code. I put a lot of efforts into minimizing patches while working on 2.3 and 2.5.

Reply:
This is a mistake. I was trying to revert my changes as it was not needed anymore.